diff --git a/node_modules/undici/lib/web/fetch/body.js b/node_modules/undici/lib/web/fetch/body.js
index 7f1819d..dc3a4e0 100644
--- a/node_modules/undici/lib/web/fetch/body.js
+++ b/node_modules/undici/lib/web/fetch/body.js
@@ -127,7 +127,11 @@ function extractBody (object, keepalive = false) {
 
     /*! formdata-polyfill. MIT License. Jimmy Wärting <https://jimmy.warting.se/opensource> */
     const escape = (str) =>
-      str.replace(/\n/g, '%0A').replace(/\r/g, '%0D').replace(/"/g, '%22')
+      str
+        .replace(/\\/g, '%5C')
+        .replace(/\n/g, '%0A')
+        .replace(/\r/g, '%0D')
+        .replace(/"/g, '%22')
     const normalizeLinefeeds = (value) => value.replace(/\r?\n|\r/g, '\r\n')
 
     // Set action to this step: run the multipart/form-data
